backend/app/api/v1/health.py

```
@@ -1,34 +1,32 @@
-    from main import DevRAIApplication
+from fastapi import APIRouter, HTTPException, Request
+from app.core.config import settings
 
 router = APIRouter()
 logger = logging.getLogger(__name__)
-
+services={}
 
 @router.get("/health")
-async def health_check(app_instance: "DevRAIApplication" = Depends(get_app_instance)):
+async def health_check(request: Request):
```

Ruff (0.14.14)

Unused function argument: `request`

(ARG001)